run_sga_tests.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+
+source /edx/app/edxapp/venvs/edxapp/bin/activate
+
+cd /edx/app/edxapp/edx-platform
+
+# these pip install commands are adapted from edx-platform/circle.yml
+pip install --exists-action w -r requirements/edx/paver.txt
+
+# Mirror what paver install_prereqs does.
+# After a successful build, CircleCI will
+# cache the virtualenv at that state, so that
+# the next build will not need to install them
+# from scratch again.
+pip install --exists-action w -r requirements/edx/pre.txt
+pip install --exists-action w -r requirements/edx/github.txt
+pip install --exists-action w -r requirements/edx/local.txt
+
+# HACK: within base.txt stevedore had a
+# dependency on a version range of pbr.
+# Install a version which falls within that range.
+pip install  --exists-action w pbr==0.9.0
+pip install --exists-action w -r requirements/edx/django.txt
+pip install --exists-action w -r requirements/edx/base.txt
+pip install --exists-action w -r requirements/edx/paver.txt
+pip install --exists-action w -r requirements/edx/testing.txt
+if [ -e requirements/edx/post.txt ]; then pip install --exists-action w -r requirements/edx/post.txt ; fi
+
+
+cd /edx-sga
+pip uninstall edx-sga -y
```

Comment:
Nit: To me it's clearer if we do this uninstall right before we copy the directory to lms/djangoapps. It's more obvious that we're replacing any currently-installed version of the package with the repo as it is